spirv_hlsl.cpp

```diff
@@ -1830,11 +1832,11 @@ void CompilerHLSL::emit_resources()
 		statement("");
 	}
 
-	const bool is_mesh_shader = execution.model == ExecutionModelMeshEXT;
-	if (!output_variables.empty() || !active_output_builtins.empty())
-	{
-		sort(output_variables.begin(), output_variables.end(), variable_compare);
-		require_output = !is_mesh_shader;
+        const bool is_mesh_shader = execution.model == ExecutionModelMeshEXT;
```

Comment:
Strange change from tabs to spaces which needs to be removed.

Reply:
Just happened to be a function I touched, so clangformat was applied. I've reverted.